docs/package.json

```diff
@@ -79,7 +79,6 @@
     "next": "^15.2.4",
     "nprogress": "^0.2.0",
     "postcss": "^8.5.3",
-    "prismjs": "^1.30.0",
```

Comment:
It's not used in this repo explicitly and it's coming from `@mui/internal-markdown` as a direct dependency.